scrapy/contrib/loader/processor.py

```diff
@@ -70,3 +74,254 @@ def __init__(self, separator=u' '):
 
     def __call__(self, values):
         return self.separator.join(values)
+
+
+class Strip(object):
+
+    def __init__(self, chars=None):
+        self.chars = chars
+
+    def __call__(self, values):
+        if isinstance(values, list):
+            filtered = []
+            for value in values:
+                value = value.strip(self.chars)
+                if value != '' and value is not None:
+                    filtered.append(value)
+            return filtered
+        return values.strip(self.chars)
+
+
+class TakeNth(object):
+
+    def __init__(self, pos, fallback_func=None):
+        self.pos = pos
+        self.fallback_func = fallback_func or (lambda values: None)
+
+    def __call__(self, values):
+        filtered = [v for v in values if v != '' and v is not None]
+        if self.pos >= len(filtered):
```

Comment:
it can be hard to debug if you don't know the code. It's better to let it fail instead of returning None.